modules/client/test/integration/client/methods.test.ts

```diff
@@ -266,19 +271,50 @@ describe("Client", () => {
               break;
             default:
               throw new Error(
-                "Unexpected DAO deposit step: " + JSON.stringify(step, null, 2)
+                "Unexpected DAO deposit step: " + JSON.stringify(step, null, 2),
               );
           }
         }
 
         expect(
           (
             await tokenContract.functions.balanceOf(
-              depositParams.daoAddressOrEns
+              depositParams.daoAddressOrEns,
             )
-          ).toString()
+          ).toString(),
         ).toBe("7");
       });
+
+      it("Check if dao registry has register dao permission in the dao factory", async () => {
+        const context = new Context(contextParamsLocalChain);
+        const client = new Client(context);
+        const params: IHasPermissionParams = {
+          daoAddressOrEns: deployment.managingDaoAddress,
+          who: deployment.daoRegistry.address,
+          where: deployment.daoFactory.address,
+          permission: Permissions.REGISTER_DAO_PERMISSION,
+        };
+        const hasPermission = await client.methods.hasPermission(params);
+        // TODO
+        // this permission is granted when the contracts are deployed so it must be true
+        // it should be true?
+        expect(hasPermission).toBe(false);
+      });
```

Comment:
We should add tests for things that are actually true. For example, we should check that in a DAO with an AddressList plugin, `hasPermission()` is true when the plugin wants to execute something on the DAO

Reply:
Yes, this is what I tried, but I was unable to find a permission that returns true, I tried with what you suggest and it is still returning false
```ts
        const params: IHasPermissionParams = {
          daoAddressOrEns: daoAddress,
          who: deployment.allowListPluginSetup.address,
          where: daoAddress,
          permission: Permissions.EXECUTE_PERMISSION,
        };

```

Reply:
Never mind, the who and where were reversed